src/express/outgoing/router.ts

```diff
@@ -0,0 +1,39 @@
+import { Router } from 'express';
+import wrapMiddleware from '../../utils/express';
+import ValidateRequest from '../../utils/joi';
+import * as outgoingController from './controller';
+import * as outgoingValidator from './validator.schema';
+
+const outgoingRouter: Router = Router();
+
+outgoingRouter.post(
+    '/:userId/outgoing',
```

Comment:
remove :userId in all routes
```suggestion
    '/outgoing',
```

Reply:
done